tests/networkpolicy_test.go

```diff
@@ -39,6 +41,16 @@ func assertPingFail(ip string, vmi *v1.VirtualMachineInstance) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
+func waitForNetworkPolicyDeletion(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, policyName string) {
```

Comment:
Please place it at the tail. It is easier to see the caller before the callee when reading the code.

There is also no need to pass the virtClient, just fetch it here (e.g. `virtCli, err := kubecli.GetKubevirtClient()`)

Reply:
placed up , just because the rest of the funcs are there (usually i put it in the tail),
and about the virtClient, the caller already has it, so saw that this is what other functions do in case it already exists

but will do those 2 changes.